edit-post/index.js

```diff
@@ -4,6 +4,7 @@
 import { registerCoreBlocks } from '@wordpress/block-library';
 import { render, unmountComponentAtNode } from '@wordpress/element';
 import { dispatch } from '@wordpress/data';
+import { setupHearthbeatPostLocking } from '../editor/utils/heartbeat-post-locking';
```

Comment:
We can't cross import between pacakges, this will create bundles with duplicated codes, we should expose `setupHearthbeatPostLocking` in the editor module and just import it like:

 `import { setupHearthbeatPostLocking } from '@wordpress/editor';`

Reply:
Ok, I'll try to do tat although I'm not exactly sure how to "expose setupHearthbeatPostLocking in the editor module"

Reply:
it just needs to be exported in the `index.js` of the editor module.

Reply:
got it, fixed.